lib/api_genres.dart

```diff
@@ -0,0 +1,50 @@
+import 'genre.dart';
+
+class ApiGenre {
+  final int id;
+  final String name;
+
+  ApiGenre({
+    required this.id,
+    required this.name,
+  });
+
+  factory ApiGenre.fromJson(Map<String, dynamic> json) {
+    return ApiGenre(
+      id: json['id'],
+      name: json['name'],
+    );
+  }
+
+  static List<Genre> mockGenres() {
+    return [
+      Genre(
+        id: 1,
+        name: 'Animation',
+      ),
+      Genre(
+        id: 2,
+        name: 'Family',
+      ),
+      Genre(
+        id: 3,
+        name: 'Adventure',
+      ),
+      Genre(
+        id: 4,
+        name: 'Fantasy',
+      ),
+      Genre(
+        id: 5,
+        name: 'Comedy',
+      ),
+    ];
+  }
+
+  Genre toModel() {
+    return Genre(
+      id: id,
+      name: name,
+    );
+  }
```

Comment:
Do you need this methods?

Reply:
After new genres handling in movie, the first method is no longer needed.
The second one is there to keep the separation between the final model the UI will use and receive, and the data that is parsed from somewhere else (in this case a local json but it can change in a future) and then converted to a model.

Reply:
But you don't need it, because you are getting the genres from the genres JSON, so the mockGenres method is not necessary to add it here

Reply:
Also review the name of the class, because api_genres here is not a good option, because you are getting a genre, but you don't know how to get it, it could be get it from a service, from database, etc